nodejs/README.md

```diff
@@ -2,25 +2,36 @@
 
 ## Steps
 
-1. Start a locally running `mongod` version >= 4.2 running on port 27017
-2. Install dependencies with npm
+1. Clone this repository.
+2. Navigate to the nodejs folder.
+
+   ```sh
+   cd nodejs
+   ```
+
+3. Start a locally running `mongod` version >= 4.2 running on port 27017
+4. Install the dependencies in `package.json`
 
    ```js
    npm install
    ```
 
-3. Run the `make-data-key.js` script to make a data key. If there is an
+5. Run the `make-data-key.js` script to make a data key. If there is an
    existing data key in the **encryption.\_\_keyVault** collection this script
    will not create a duplicate data key.
 
    ```js
    node make-data-key.js
    ```
 
-   You will use the output of this script to seed the base64 key id used in
-   `clients.js`
+   This outputs a base64 encoded string of your newly created data key. Paste
+   this into `clients.js` where you see this line
+
+   ```js
+   let dataKey = null // change this!
+   ```
 
-4. Run the `clients.js` script to insert a document with the CSFLE-enabled client
+6. Run the `clients.js` script to insert a document with the CSFLE-enabled client
```

Comment:
Nitpick: Could use a pronoun to avoid mentioning "CSFLE-enabled client" twice. E.g.
"... with the CSFLE-enabled client and then read/retrieve that document with it as well as with a regular client"